src/java.base/unix/classes/sun/security/provider/NativePRNG.java

```diff
@@ -385,11 +466,12 @@ private static class RandomIO {
         private final Object LOCK_SET_SEED = new Object();
 
         // constructor, called only once from initIO()
-        private RandomIO(File seedFile, File nextFile) throws IOException {
+        private RandomIO(File seedFile, File nextFile, final Variant variant) throws IOException {
```

Comment:
Please remove `final` here; it isn't needed and it's inconsistent: it's not specified for the parameter of `readFully()`.

Reply:
removed